### DISCUSSION TITLE
Support for subscriptions

### DISCUSSION THREAD

**Discussion by andrewwilkin** [RevereCRE/relay-nextjs#79] [Q&A] [answered]
Was wondering whether it is possible to use this library with subscriptions?

As would like to use this for sending notifications from server to client for things such as new messages, and this does mean that the token needs to be accessible to the client side to set up the subscription.

**Comment by andrewwilkin** [comment, marked as answer]
Found the answer in this issue:
https://github.com/RevereCRE/relay-nextjs/issues/54

:)

**Reply by rrdelaney** [reply]
I can personally confirm that it works too, we’ve started experimenting with subscriptions and had a lot of success!
